src/libs/ajax/DAC.js

```diff
@@ -0,0 +1,82 @@
+import * as fp from 'lodash/fp';
```

Comment:
Now that these are all in separate modules, it should make it easier to test. Maybe we can add a ticket in the backlog to test these?

Reply:
What kind of tests are you thinking? Would these be unit tests to verify that the requests are in the shape we are expecting (since I believe that is/should be the only logic in these ajax functions)? Or are you thinking of something else?

Reply:
In our Java apps, we use MockServer to mock the potential responses. I'm not familiar with a JS equivalent, but we should look into it.

Reply:
Another backlog ticket related to this effort is to finish migrating the `fetch` calls to `axios`.